pmoves/design/build/tokens.pmoves-armor.css

```
@@ -0,0 +1,27 @@
+/* generated by pmoves/design/generate.py — do not edit */
+:root[data-theme="pmoves-armor"], :root:not([data-theme]) {
+  --pm-accent: #7C3AED;
+  --pm-accent-2: #0D9488;
+  --pm-accent-soft: #A78BFA;
+  --pm-bg: #050508;
+  --pm-bg-tint: #0a0a12;
+  --pm-border-strong: rgba(255,255,255,0.15);
+  --pm-border-subtle: rgba(255,255,255,0.08);
+  --pm-font-body: 'Exo 2', system-ui, sans-serif;
+  --pm-font-display: Orbitron, 'Exo 2', system-ui, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "Orbitron" to be "orbitron" (value-keyword-case)

(value-keyword-case)